week3/homework/src/actions.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+const fs = require('fs');
```

Comment:
This file does not seem to be used :(

Reply:
Hi @gajduk, thanks for the feedback and for your time. The changes have been made.